src/lib/modal/Modal.svelte

```
@@ -5,100 +5,109 @@
+    close(dlg);
+  }
+
+  function _ontoggle(ev: ToggleEvent & { currentTarget: HTMLDialogElement }) {
+    ontoggle?.(ev);
+    open = ev.newState === "open"; // for cases when toggle by other means
   }
 
   function init(dlg: HTMLDialogElement) {
-    modal ? dlg?.showModal() : dlg?.show();
-    return () => dlg?.close();
+    modal ? dlg.showModal() : dlg.show();
```

ESLint

[error] Expected an assignment or function call and instead saw an expression.

(@typescript-eslint/no-unused-expressions)